src/components/matching/CandidatesStack.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from "react";
+
+const CandidatesStack = () => (
+  <div>
+    <h1>Candidates Stack</h1>
+    <p>Candidate Stack placeholder</p>
+  </div>
+);
+
+export { CandidatesStack };
```

Comment:
Consider using `export const` instead of a separate export statement

Reply:
ok